src/Common/Update.php

```diff
@@ -119,12 +119,13 @@ public function orWhere($cond)
      *
      * @param string $col The column name.
      *
-     * @return $this
+     * @param array  $value
```

Comment:
there is extra space, may be you could remove it also for the sake of consistency ?